lib/rest-api.php

```diff
@@ -277,6 +277,24 @@ function gutenberg_add_target_schema_to_links( $response, $post, $request ) {
 	$new_links  = array();
 	$orig_links = $response->get_links();
 	$post_type  = get_post_type_object( $post->post_type );
+	if ( 'edit' === $request['context'] && post_type_supports( $post_type->name, 'author' ) ) {
+		if ( current_user_can( $post_type->cap->edit_others_posts ) ) {
+			$new_links['https://api.w.org/action-assign-author'] = array(
+				array(
+					'title'        => __( 'The current user change author on this post.', 'gutenberg' ),
```

Comment:
A little grammar tweak: `'The current user can change the author on this post.'`

Reply:
Updated in b13fe8009914dd01f6fa31dc8784d127d168785c